superset-frontend/src/filters/components/Range/RangeFilterPlugin.tsx

```diff
@@ -16,12 +16,62 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import { getNumberFormatter, NumberFormats, t } from '@superset-ui/core';
+import {
+  getNumberFormatter,
+  NumberFormats,
+  styled,
+  t,
+} from '@superset-ui/core';
 import React, { useEffect, useState } from 'react';
 import { Slider } from 'src/common/components';
 import { PluginFilterRangeProps } from './types';
 import { Styles } from '../common';
 import { getRangeExtraFormData } from '../../utils';
+import FormItem from '../../../components/Form/FormItem';
+
+const Error = styled.div`
+  color: ${({ theme }) => theme.colors.error.base};
+`;
+
+const Wrapper = styled.div<{ validateStatus?: string }>`
+  border: 1px solid transparent;
+  &:focus {
+    border: 1px solid
+      ${({ theme, validateStatus }) =>
+        validateStatus ? theme.colors.error.base : theme.colors.primary.base};
+    outline: 0;
+    box-shadow: 0 0 0 5px
+      ${({ validateStatus }) =>
+        validateStatus ? 'rgba(224, 67, 85, 12%)' : 'rgba(224, 67, 85, 12%)'};
+  }
+  & .ant-slider {
+    & .ant-slider-track {
+      background-color: ${({ theme, validateStatus }) =>
+        validateStatus && theme.colors.error.light1};
+    }
+    & .ant-slider-handle {
+      border: ${({ theme, validateStatus }) =>
+        validateStatus && `2px solid ${theme.colors.error.light1}`};
+      &:focus {
+        box-shadow: 0 0 0 5px
+          ${({ validateStatus }) =>
+            validateStatus
+              ? 'rgba(224, 67, 85, 12%)'
+              : 'rgba(32, 167, 201, 0.2)'};
+      }
+    }
+    &:hover {
+      & .ant-slider-track {
+        background-color: ${({ theme, validateStatus }) =>
+          validateStatus && theme.colors.error.base};
+      }
+      & .ant-slider-handle {
+        border: ${({ theme, validateStatus }) =>
+          validateStatus && `2px solid ${theme.colors.error.base}`};
+      }
+    }
+  }
```

Comment:
It seems like these validation status color changes weren't being picked up (I believe the same applies to time filter):

![image](https://user-images.githubusercontent.com/33317356/122766035-be5cf080-d2a9-11eb-9b05-d78d91f2b4d4.png)